xls/modules/zstd/frame_header_test.cc

```diff
@@ -113,7 +126,7 @@ class FrameHeaderTest : public xls::IrTestBase {
   // expected values.
   void RunAndExpectFrameHeader(const ZstdFrameHeader& zstd_frame_header) {
     // Extend buffer contents to 128 bits if necessary.
-    const absl::Span<const uint8_t>& buffer = zstd_frame_header.kBuffer;
+    const absl::Span<const uint8_t>& buffer = zstd_frame_header.kBuffer();
```

Comment:
Don't take refs to spans, spans are easily copied so we pass them and hold them by value

Reply:
Fixed